yb-voyager/src/srcdb/pg_dump_extract_schema.go

```diff
@@ -135,7 +135,14 @@ func parseSchemaFile(exportDir string, schemaDir string, exportObjectTypesList [
 				objSqlStmts["SEQUENCE"].WriteString(stmts)
 			case "INDEX", "INDEX ATTACH":
 				objSqlStmts["INDEX"].WriteString(stmts)
-			case "TABLE", "DEFAULT", "CONSTRAINT", "FK CONSTRAINT", "CHECK CONSTRAINT", "ROW SECURITY":
+			case  "DEFAULT":
+				//In cases the DEFAULT stmt is for the FOREIGN TABLE, it should go in foreign_table.sql
```

Comment:
Are we making the change to write to "foreign_table.sql" in this PR? 
because this could be the case for other possible ALTERs as well, right? (not just for SET DEFAULT) https://www.postgresql.org/docs/current/sql-alterforeigntable.html

Reply:
> Are we making the change to write to "foreign_table.sql" in this PR?

No, it was already earlier itself we are writing the CREATE FOREIGN TABLEs  or the ones which will have the `FOREIGN TABLE` as type by pg_dump into that file. I believe the other type of ALTER FOREIGN TABLEs will have the type as `FOREIGN TABLE` similar to `TABLE`. But yes, these are some common types like DEFAULT, CONSTRAINT etc..
I will check what else need to be fixed as well.

Reply:
>, "CONSTRAINT", "FK CONSTRAINT", "CHECK CONSTRAINT", "ROW SECURITY"

out of these types, none of them is possible as only Check constraint is supported on foreign tables which is coming up in CREATE FOREIGN DDL. 

So we should be good to only fixing the DEFAULT type to be added as per ALTER FOREIGN TABLE clause.

Reply:
Got it, thanks!